examples/sqlite_drizzle/test/test.ts

```diff
@@ -0,0 +1,7 @@
+import { getCanisterId } from 'azle/dfx';
+import { getTests } from 'azle/examples/sqlite/test/tests';
```

Comment:
So it turns out this actually is a problem in CI/CD with tests, who would have known right?

Reply:
I have just addressed this and it works locally.